src/components/common/contact-us-form/index.tsx

```diff
@@ -75,7 +75,7 @@ const ContactForm: React.FC = () => {
     try {
       setLoading(true);
       const response = await fetch(
-        "https://test.gracefilledcollege.com/public/api/v1/contact",
+        `${process.env.NEXT_PUBLIC_BACKEND_PROBE_URL}/api/v1/contact`,
```

Comment:
This would break the application, what exactly is your plan, there's a utility function getApiUrl already